lib/bfd.c

```diff
@@ -1352,17 +1333,6 @@ static bool bfd_source_cache_update(struct bfd_source_cache *source,
 	return false;
 }
 
-void bfd_nht_zclient_connected(struct zclient *zclient)
```

Comment:
Removing this breaks the source detection on zebra client reconnect.

Reply:
it does not because nht (un)registration is done in staticd and bfd_nht_update() deals with the data